### PR TITLE
fix(web): improve 2fa enrollment process

### LINKED ISSUE #1560: 2FA enrollment is difficult to understand.
Related: #135

 - Rename OTP to TOTP. OTP can mean a wide variety of things.
 - Specify to the user, what security key standards are meant. U2F iirc, maybe it was WebAuthn?
Maybe subtexts explaining on what it is?
 - #1559 allow multiple tokens, and make it clear what tokens (name them!) are added, etc. I discovered multiple tokens weren't available only through testing. If a user registers 2 tokens, they could lock themselves out for some period.

The process of registering a new token was quite finicky as well. It's like buttons without any context.

### PR DESCRIPTION
**Pull request by nightah**
This PR will change some of the wording and colours for the 2FA processes in order to provide more clarity and address some accessibility issues for end users.

The following is a summary of the changes:

* One-Time Password ⭢ Time-based One-Time Password
* Security Key ⭢ Security Key - U2F

Screenshot_2021-02-02-09-36-17

* QRCode ⭢ QR Code

Screenshot_2021-02-07-05-07-25

* `Not registered yet?` text to display `Lost device?` if a user has already registered a device of said type

Screenshot_2021-02-02-10-24-54

* Change button and text colour in e-mails that Authelia generates
* Change Authelia email footer to be more security conscious

Screenshot_2021-02-07-04-51-40

The docs have also been updated to clarify the 2fa device enrollment limitation which only allows users to register one of each device type concurrently.

Closes #1560.

### CODE REVIEW

File: docs/features/2fa/security-key.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ by simply touching the token again when requested:
 
 Easy, right?!
 
+
+## Limitation
```
Comment (james-d-elliott) [inline docs/features/2fa/security-key.md:48]:
```suggestion
## Limitations
```

File: internal/suites/scenario_available_methods_test.go [hunk 2]
```diff
@@ -48,16 +50,6 @@ func (s *AvailableMethodsScenario) SetupTest() {
...
-func IsStringInList(str string, list []string) bool {
-	for _, v := range list {
-		if v == str {
-			return true
-		}
-	}
-
-	return false
-}
-
```
Comment (james-d-elliott) [inline internal/suites/scenario_available_methods_test.go:60]: Nice!

File: internal/templates/plaintext_email.go [hunk 3]
```diff
@@ -22,5 +22,5 @@ If you did not initiate the process your credentials might have been compromised
 
 To setup your 2FA please visit the following URL: {{.url}}
 
-Please ignore this email if you did not initiate the process.
+Please contact an administrator if you did not initiate the process.
```
Comment (james-d-elliott) [inline internal/templates/plaintext_email.go:25]: Yeah, that's much better.

**Review by james-d-elliott — APPROVED** [review]
LGTM

File: internal/suites/scenario_available_methods_test.go (hunk 2 above)
Comment (james-d-elliott) [reply internal/suites/scenario_available_methods_test.go:60]: (removing duplicate func)

File: docs/features/2fa/one-time-password.md [hunk 4]
```diff
@@ -34,5 +33,9 @@ From now on, you get tokens generated every 30 seconds that
 you can use to validate the second factor in **Authelia**.
 
 
+## Limitation
```
Comment (nightah) [inline docs/features/2fa/one-time-password.md:36]:
```suggestion
## Limitations
```

**Review by james-d-elliott — APPROVED** [review]

**Pull request merged**